### ISSUE TITLE
[BUG/ISSUE] (12.7.2) Invalid longitude subset values for collection "BoundaryConditions"!

### ISSUE THREAD

**Issue by Holton1** [geoschem/geos-chem#1175]
### Ask a question about GEOS-Chem:

Hi, I am using GEOS-Chem 12.7.2 global 2x2.5 simulation to output boundary condition fields. To save storage space, I set the boundarycondition domain to be northern hemisphere ([-180,180],[0,90]) in HISTORY.rc.   

My setting in HISTORY.rc is:

```console
563 #==============================================================================
564 # %%%%% THE BoundaryConditions COLLECTION %%%%%
565 #
566 # GEOS-Chem boundary conditions for use in nested grid simulations
567 #
568 # Available for all simulations
569 #==============================================================================
570   BoundaryConditions.template:   '%y4%m2%d2_%h2%n2z.nc4',
571   BoundaryConditions.format:     'CFIO',
572   BoundaryConditions.frequency:  00000000 030000
573   BoundaryConditions.duration:   00000001 000000
574   BoundaryConditions.mode:       'instantaneous'
575   BoundaryConditions.LON_RANGE:  -180.0 180.0,
576   BoundaryConditions.LAT_RANGE:  0.0 90.0,
577   BoundaryConditions.fields:     'SpeciesBC_?ADV?             ', 'GIGCchem',
```
The error report says:

```console
1895 ===============================================================================
1896 GEOS-Chem ERROR: Invalid longitude subset values for collection "BoundaryConditions"!                                                                                       
1897  -> at History_ReadCollectionData (in module History/history_mod.F90)
1898 
1899  -> ERROR occurred at (or near) line    575 of the HISTORY.rc file
1900 ===============================================================================
1901 
```

I tried setting the boundary condition output domain to be smaller ([-180,-120],[40,85]), and the model runs well and doesn't crash.

I wonder is there any limitation on the domain size of the boundary condition subset?  Thanks a lot!

**Comment by yantosca** [comment]
Thanks for writing @Holton1. We issued a fix for this that went into 13.3.4.  See #1045 and #1047 for details.

**Comment by Holton1** [comment]
Thanks Bob!
